app/models/profile.rb

```diff
@@ -79,29 +72,19 @@ def fill_from_parent
     self
   end
 
-  def canonical?
-    parent_profile_id.blank?
-  end
-
-  def clone_to(account:, policy:, os_minor_version: nil)
-    new_profile = in_account(account, policy, os_minor_version)
+  def clone_to(account:, policy:,
+               os_minor_version: nil, set_os_minor_version: nil)
+    new_profile = in_account(
+      account, policy,
+      os_minor_version || set_os_minor_version
```

Comment:
This is a bit confusing, but I have no better idea on how to name this :thinking: